JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsRoute.kt

```diff
@@ -28,14 +28,14 @@ import androidx.compose.runtime.saveable.rememberSaveable
  * @param interestsViewModel ViewModel that handles the business logic of this screen
  * @param isExpandedScreen (state) true if the screen is expanded
  * @param openDrawer (event) request opening the app drawer
- * @param scaffoldState (state) state for screen Scaffold
+ * @param snackbarHostState (state) state for screen snackbar host
  */
 @Composable
 fun InterestsRoute(
     interestsViewModel: InterestsViewModel,
     isExpandedScreen: Boolean,
     openDrawer: () -> Unit,
-    scaffoldState: ScaffoldState = rememberScaffoldState()
+    snackbarHostState: SnackbarHostState = remember { SnackbarHostState() }
```

Comment:
Should we have a `rememberSnackbarHostState` method defined? I see this pattern used elsewhere

Reply:
I don't know? It's not part of the public API, so I just used what's being used elsewhere. Something to discuss with @astamato maybe?

Reply:
Interesting, I'd leave as is for now. All the rememberX functions ive come across do smth more complex than just remember a single value.